src/components/letterBoxComponents/NoLetter.tsx

```
@@ -4,11 +4,12 @@ import React from 'react';
 import Text from '../common/Text';
 import Image from 'next/image';
 import Link from 'next/link';
+import { formatDate } from '@/app/utils/letterbox/dateUtils';
```

ESLint

[error] 'formatDate' is defined but never used.

(@typescript-eslint/no-unused-vars)